common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -539,7 +550,10 @@ def should_show_save_button(self):
             # then do NOT show the save button
             # If we're waiting for the user to reset a randomized problem
             # then do NOT show the save button
-            elif (self.closed() and not is_survey_question) or needs_reset:
+            # If the question is timed, enforce that we can only attempt
+            # once. Thus, do NOT show the save button
+            elif (self.closed() and not is_survey_question) or needs_reset \
+                    or submitted_timed_question:
```

Comment:
simple style thing: lines can be 120 characters wide, and the use of `\` continuation is discouraged, so probably can just move this line up